integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -928,6 +934,155 @@ public static void createDirUnderDomainPV(String dirPath) throws Exception {
     logger.info("command result " + result.stdout().trim());
   }
 
+  /**
+   * A utility method to copy Cross Namespaces RBAC yaml template file replacing the DOMAIN_NS,
+   * OPERATOR_NS
+   *
+   * @throws IOException when copying files from source location to staging area fails
+   */
+  private static void createCrossNSRBACFile(String domainNS, String operatorNS) throws IOException {
+    String samplesDir = monitoringExporterDir + "/src/samples/kubernetes/";
+    Path src = Paths.get(samplesDir + "/crossnsrbac.yaml");
+    Path dst = Paths.get(samplesDir + "/crossnsrbac_" + domainNS + "_" + operatorNS + ".yaml");
+    if (!dst.toFile().exists()) {
+      logger.log(Level.INFO, "Copying {0}", src.toString());
+      Charset charset = StandardCharsets.UTF_8;
+      String content = new String(Files.readAllBytes(src), charset);
+      content = content.replaceAll("weblogic-domain", domainNS);
+      content = content.replaceAll("weblogic-operator", operatorNS);
+      logger.log(Level.INFO, "to {0}", dst.toString());
+      Files.write(dst, content.getBytes(charset));
+    }
+  }
+
+  /**
+   * A utility method to add desired domain namespace to coordinator yaml template file replacing
+   * the DOMAIN_NS,
+   *
+   * @throws IOException when copying files from source location to staging area fails
+   */
+  private static void createCoordinatorFile(String domainNS) throws IOException {
+    // String coordinatorDir = BaseTest.getProjectRoot() +
+    // "/integration-tests/src/test/resources/exporter/";
+    Path src = Paths.get(resourceExporterDir + "/coordinator.yml");
+    Path dst = Paths.get(resourceExporterDir + "/coordinator_" + domainNS + ".yaml");
+    if (!dst.toFile().exists()) {
+      logger.log(Level.INFO, "Copying {0}", src.toString());
+      Charset charset = StandardCharsets.UTF_8;
+      String content = new String(Files.readAllBytes(src), charset);
+      content = content.replaceAll("default", domainNS);
+      logger.log(Level.INFO, "to {0}", dst.toString());
+      Files.write(dst, content.getBytes(charset));
+    }
+  }
+
+  /**
+   * Remove monitoring exporter directory if exists and clone latest from github for monitoring
+   * exporter code
+   *
+   * @throws Exception if could not run the command successfully to clone from github
+   */
+  public static void gitCloneBuildMonitoringExporter() throws Exception {
```

Comment:
If these methods are used only in Monitoring Exporter testing, you can move them to the IT class. Also please add more comments inside the code.

Reply:
moved all to ITMonitoringExporter